From 7294d1dbef227006979ba286c41607edbc8f83ff Mon Sep 17 00:00:00 2001
From: Megha Dey <megha.dey@linux.intel.com>
Date: Wed, 2 Nov 2016 17:16:30 -0700
Subject: [PATCH] pstore : start after network

We have to make sure pstore probe starts only once network is up, else the
pstore records will not be sent to the server.

Signed-off-by: Megha Dey <megha.dey@linux.intel.com>
---
 src/data/pstore-probe.service.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/data/pstore-probe.service.in b/src/data/pstore-probe.service.in
index 41e4643..fe71262 100644
--- a/src/data/pstore-probe.service.in
+++ b/src/data/pstore-probe.service.in
@@ -1,7 +1,8 @@
 [Unit]
 Description=Telemetrics Pstore Probe
 Requires=telemd.socket
-After=telemd.socket
+Wants=networking.service
+After=telemd.socket networking.service

 [Service]
 ExecStart=@bindir@/pstoreprobe
--
1.9.1
